opentelemetry-semantic-conventions/CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 ## vNext
 
+## v0.26.0
+### Changed
+
+- Starting with this version, this crate will use Weaver for the generation of
+  the semantic conventions.
+- Introduced a new feature `semconv_experimental` to enable experimental semantic conventions.
```

Comment:
This is breaking change ? Just prefix it with  `**Breaking**` if so ?

Reply:
Fixed in [4970ee9](https://github.com/open-telemetry/opentelemetry-rust/pull/2098/commits/4970ee9314ca5cb72376b21bb807ec14da0d4cc8)